nl-core/StructureDefinitions/ext-CopyIndicator.xml

```diff
@@ -0,0 +1,82 @@
+  </text>
+  <url value="http://nictiz.nl/fhir/StructureDefinition/ext-CopyIndicator" />
+  <name value="ExtCopyIndicator" />
+  <title value="ext CopyIndicator" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="url" />
+      <value value="https://www.nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="This extension provides an indication whether the resource is original (false) or a copy from a different source (true)." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="lu-dataset-laboratory-exchange-20210222" />
+    <uri value="https://decor.nictiz.nl/ad/#/lu-/datasets/dataset/2.16.840.1.113883.2.4.3.11.60.25.1.2/2021-02-22T00:00:00" />
+    <name value="ART-DECOR Dataset Lab Exchange 20210222T00:00:00" />
+  </mapping>
+  <kind value="complex-type" />
+  <abstract value="false" />
+  <context>
+    <type value="element" />
+    <expression value="MedicationStatement" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="MedicationDispense" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="Observation" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="Specimen" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="Device" />
+  </context>
+  <context>
+    <type value="element" />
+    <expression value="AllergyIntolerance.reaction" />
+  </context>
+  <type value="Extension" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Extension" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Extension">
+      <path value="Extension" />
+      <max value="1" />
+    </element>
+    <element id="Extension.url">
+      <path value="Extension.url" />
+      <fixedUri value="http://nictiz.nl/fhir/StructureDefinition/ext-CopyIndicator" />
+    </element>
+    <element id="Extension.value[x]">
+      <path value="Extension.value[x]" />
+      <short value="CopyIndicator" />
+      <definition value="An indication whether the resource is original (false) or a copy from a different source (true)." />
+      <alias value="KopieIndicator" />
+      <type>
+        <code value="boolean" />
+      </type>
+      <mapping>
+        <identity value="lu-dataset-laboratory-exchange-20210222" />
```

Comment:
Mm, I think we should remove this mapping. Or perhaps remove this extension all together. We will get this zib in 2024: https://zibs.nl/wiki/RegistrationData-v1.1.1(2024EN) ->RegistrationData.CopyIndicator